pages/common/gh-auth.md

```diff
@@ -0,0 +1,24 @@
+# gh-auth
+
+> Authenticate with GitHub in GitHub cli.
+> More information: <https://cli.github.com/manual/gh_auth_login>.
+
+- Login with interactive prompt:
+
+`gh auth login`
+
+- Login with token from https://github.com/settings/tokens :
+
+`echo {{your_token}} | gh auth login --with-token`
+
+- Check if you are logged in:
+
+`gh auth status`
+
+- Log out:
```

Comment:
```suggestion
- Log out from a GitHub host interactively:
```

Reply:
Again here, I would keep it short - yes, it is interactive, but like, everything that you do in `gh` is, so it's kidna obvious

Reply:
:thinking: not for me... I'm logged in just 1 account and it's interactively too.

Reply:
Yes, that's what I ment - it asks you if you want to log out `yes/no`. But every command in `gh` is interactive, if you won't give it **all** the data in parameters, so it's kidna obvious, and I would keep this short to make this more readable :+1: